packages/uikit/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -39,21 +39,45 @@ const DropdownMenu: React.FC<React.PropsWithChildren<DropdownMenuProps>> = ({
   const isMenuShow = isOpen && ((isBottomNav && showItemsOnMobile) || !isBottomNav);
 
   useEffect(() => {
+    let timeoutId: NodeJS.Timeout | null = null;
```

Comment:
Let's use `ReturnType<typeof setTimeout>` for the type of `timeoutId` cuz it's actually run on browser